interface/main/backup.php

```diff
@@ -814,19 +814,35 @@ function export_submit(step) {
                     echo xlt("Skipping missing layout name") . ": " . text($layoutid) . "<br>";
                     continue;
                 }
-                // Windows uses the & to join statements.
-                $cmdsep = IS_WINDOWS ? ' & ' : ';';
-                $cmd .= "echo 'DELETE FROM layout_options WHERE form_id = \"" . add_escape_custom($layoutid) . "\";'" .
-                    " >> " . escapeshellarg($EXPORT_FILE) . $cmdsep;
-                $cmd .= "echo 'DELETE FROM layout_group_properties WHERE grp_form_id = \"" . add_escape_custom($layoutid) . "\";'" .
-                    " >> " . escapeshellarg($EXPORT_FILE) . $cmdsep;
-                $cmd .= $dumppfx . " --where='grp_form_id = \"" . add_escape_custom($layoutid) . "\"' " .
-                    escapeshellarg($sqlconf["dbase"]) . " layout_group_properties" .
-                    " >> " . escapeshellarg($EXPORT_FILE) . $cmdsep;
-                $cmd .= $dumppfx . " --where='form_id = \"" . add_escape_custom($layoutid) . "\" ORDER BY group_id, seq, title' " .
-                    escapeshellarg($sqlconf["dbase"]) . " layout_options" .
-                    " >> " . escapeshellarg($EXPORT_FILE) . $cmdsep;
-
+                // Beware and keep in mind that Windows requires double quotes around arguments.
+                if (IS_WINDOWS) {
+                    # windows will place the quotes in the outputted code if they are there. we removed them here.
+                    $cmd .= " echo 'DELETE FROM layout_options WHERE form_id = \"" . add_escape_custom($layoutid) . "\";' >> " . escapeshellarg($EXPORT_FILE) . " & ";
+                } else {
+                    $cmd .= "echo 'DELETE FROM layout_options WHERE form_id = \"" . add_escape_custom($layoutid) . "\";' >> " . escapeshellarg($EXPORT_FILE) . ";";
+                }
+                if (IS_WINDOWS) {
+                    # windows will place the quotes in the outputted code if they are there. we removed them here.
+                    $cmd .= "echo 'DELETE FROM layout_group_properties WHERE grp_form_id = \"" . add_escape_custom($layoutid) . "\";' >> " . escapeshellarg($EXPORT_FILE) . " &;";
```

Comment:
Is " &;" at the end intentional here, or should it be " & " ?

Reply:
Nope, correct as is. Thanks for looking at though.

Reply:
I do have to figure out where new required dependency layout.inc.php came from and whether i'll be able to bring back for patch 2.

Reply:
It's needed for CSV export of layouts, which is not in 6.0.0.

Reply:
Do we want it in 6 or leave it be as current backup for 6.0 works fine? I can bring back if wanted

Reply:
I would not add it to 6... most new features go into master only.